GoogleChrome/developer.chrome.com
# Add multiple authors design

Fixes #6009

## Changes proposed in this pull request:
- When there are 3 or more authors on a page, collapse the design to look similar to cards (condensed design)
- Add a new css class called 'post-authors__image' which gives the images a 50px width and size rather than a 40px

## How to test the change
1. Clone the repository, checkout the branch and build the site
2. Head to /blog/test-automation-evolution/, this will show the two authors in their expected showcase
3. Head to /docs/privacy-sandbox/glossary/, this will show the three authors in the new condensed design
